code/datums/weather/weather_types.dm

```diff
@@ -134,20 +140,33 @@
 
 	immunity_type = "rad"
 
+/datum/weather/rad_storm/telegraph()
+	rad_strength = rand(rad_strength_lower, rad_strength_upper)
+	delay = ((weather_duration - 100) / rad_strength)
+	..()
+
 /datum/weather/rad_storm/impact(mob/living/L)
-	if(ishuman(L))
-		var/mob/living/carbon/human/H = L
-		if(H.dna && H.dna.species)
-			if(!(RADIMMUNE in H.dna.species.specflags))
-				if(prob(25))
+	if(armed)
+		if(ishuman(L))
+			var/mob/living/carbon/human/H = L
+			if(H.dna && H.dna.species)
+				if(!(RADIMMUNE in H.dna.species.specflags))
 					if(prob(25))
-						randmuti(H)
-					if(prob(90))
-						randmutb(H)
-					else
-						randmutg(H)
-					H.domutcheck()
-	L.rad_act(20,1)
+						if(prob(25))
+							randmuti(H)
+						if(prob(90))
+							randmutb(H)
+						else
+							randmutg(H)
+						H.domutcheck()
+		L.rad_act(20,1)
+		armed = 0
```

Comment:
Oh well that sucks. >Weather doesn't have an innate cooldown between impact calls